src/content/en/updates/2017/06/play-request-was-interrupted.md

```diff
@@ -0,0 +1,146 @@
+<blockquote class="warning">
+Uncaught (in promise) DOMException: The play() request was interrupted by a
+call to pause().
+</blockquote>
+
+You're in the right place then. Have no fear. I'll explain [what is causing
+this](#error) and [how to fix it](#fix).
+
+## What is causing this {: #error }
+
+Here's some JavaScript code below that reproduces the "uncaught (in promise)"
+error you're seeing:
+
+<span class="compare-worse">DON'T</span>
+```html
+<video id="video" preload="none" src="https://example.com/file.mp4"></video>
+
+<script>
+  video.play(); // <-- WRONG! This is NOT synchronous.
+  video.pause();
+</script>
+```
+
+The code above results in this error message in Chrome DevTools: 
+
+> _Uncaught (in promise) DOMException: The play() request was interrupted by a
+> call to pause()._
+
+This is due to the fact that the `play()` call is NOT synchronous, like it used
+to be. Since Chrome 50 - [March 2016], a `play()` call on an a `<video>` or
+`<audio>` element returns a [Promise], a function that returns a single result
+**asynchronously**. If playback succeeds, the Promise is fulfilled, and if
+playback fails, the Promise is rejected along with an error message explaining
+the failure.
+
+As `video.play()` is asynchronous, `video.pause()` is executed before
+video playback actually starts. This causes `video.play()` to return a rejected
+Promise as user just asked to pause video. Think about it this way:
+`video.play()` fails eventually.
+
+Note: Calling `video.pause()` isn't the only way to reset video playback state.
+It also occurs with `video.src = ''`, `video.removeAttribute('src')`, and
+`video.load()` for instance.
+
+Since we're not handling the video play Promise in our code, an error message
+appears in Chrome DevTools. This is what is really happening...
+
+## How to fix it {: #fix }
+
+Now that we understand the root cause, let's see what we can do to fix this
+properly.
+
+First, don't ever assume a media element (video or audio) will play. Look at
+the Promise returned by the `play` function to see if it was rejected. It is
+worth noting that the Promise won't fulfill until playback has actually
+started, meaning the code inside the `then()` will not execute until the media
+is playing.
+
+<span class="compare-better">Example: Play & Pause</span>
+```html
+<video id="video" preload="none" src="https://example.com/file.mp4"></video>
+
+<script>
+  var playPromise = video.play();
+
+  playPromise.then(_ => {
+    // Automatic playback started!
+    // We can now pause video...
+    video.pause();
+  })
+  .catch(error => {
+    // Automatic playback failed.
+    // This can be due to an autoplay policy on mobile for instance...
+  });
+</script>
+```
+
+That's great for this simple example but what if you use `video.play()` to be
+able to play a video later when user interacts with the website you may think?
+
+I'll tell you a secret... you don't have to use `video.play()`, you should use
+`video.load()`. And here's how:
```

Comment:
I'll update this article when new autoplay policy is out.
It will still be useful for Safari though.